src/components/single-article.jsx

```diff
@@ -0,0 +1,75 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import getArticles from "../api";
+
+function SingleArticle() {
+  const [singleArticle, setSingleArticle] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  let { article_id } = useParams();
+
+  console.log(typeof article_id);
+
+  useEffect(() => {
+    getArticles().then((data) => {
```

Comment:
Rather than making the same api call, we should be bringing in another function that brings the data for the single article by ID. You then won't have to use the .find method below.